src/styles/components/_checkbox.scss

```diff
@@ -0,0 +1,30 @@
+@import '../settings/colors';
+
+.w-checkbox__label {
+  cursor: pointer;
+  display: inline-flex;
+  position: relative;
+}
+
+.w-checkbox__input {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+  background: url('/images/icons/check_box_outline_blank.svg');
+  border: 2px solid $WHITE;
+  border-radius: 4px;
+  cursor: pointer;
+  height: 28px;
+  margin: 0px 30px 0px 0px;
```

Comment:
suggest changing this to something like `margin: 3px` or some default value that wraps all the way around, similar to the native checkbox.

ideally components shouldn't define their own margins, they should be set by the component or page that's using them. otherwise you end up needing a lot of overrides.

I think in this case, having a default margin that wraps all the way around, or no margin at all, seem ok as both of those would match the expectations for a checkbox. But just having it on one side feels a little too bespoke to how we're using it on this particular page.